bundle-all.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var argv = require('minimist')(process.argv.slice(2));
+const fs = require('fs');
+const path = require('path');
+const exec = require('child_process').exec;
+
+fs.readdirSync(argv['i']).forEach(dir => {
+    if (!fs.existsSync(`${argv['o']}/${dir}`)) {
+        fs.mkdirSync(`${argv['o']}/${dir}`);
+    }
+
+    exec(`./node_modules/browserify/bin/cmd.js ${argv['i']}/${dir}/index.js -o ${argv['o']}/${dir}/index.js -x jquery`, function(error, stdout, stderr) {
+        if (error) {
+            console.log(error)

```

Comment:
this should fail the build right? we can use `process.exit(1)` and `console.error`